sematic/resolvers/local_resolver.py

```diff
@@ -220,6 +222,7 @@ def _create_resolution(self, root_future):
         """Make a Resolution instance and persist it."""
         resolution = self._make_resolution(root_future)
         api_client.save_resolution(resolution)
+        self._notify_pipeline_update()
```

Comment:
Just some DRY instead of `_future_did_schedule` below.

Reply:
No this does not change the behavior. Now the notification is shown upon creating of the resolution, which means that the root run already exist. If anything, this will get the notification fired slightly earlier, which makes the experience more snappy.

This is a more logical place to put it and removes the need to override `_future_did_schedule` and do some inelegant logic to only fire for the root run.